event-store-client/readmodels.md

```diff
@@ -15,7 +15,7 @@ c) Write a script that subscribes to it.
 * * *
 
 Your event streams should be organized as one stream per aggregate. If you have 5 users in your system and the aggregate ids are simply `1`, `2`, `3`, `4`, `5`, then you may have 5 streams with these names:
-`user-1`, `user-2`, `user-3`, `user-4`, `user-5`.
+`user-1`, `user-2`, `user-3`, `user-4`, `user-5`. Your stream name (before id) **cannot contain dashes** because eventstore use them with this patern `aggregateCategory-{id}` (id can contain dashes).
```

Comment:
1) typo: "pattern" not "patern".
2) using "dash" is only the default, it's something you can configure, f.e. you can use "." instead, if you prefer that.

Reply:
for configuration see here: https://eventstore.com/docs/projections/system-projections/index.html?tabs=tabid-5